mitmproxy/io/tnetstring.py

```diff
@@ -202,7 +202,7 @@ def parse(data_type: int, data: bytes) -> TSerializable:
     if data_type == ord(b"!"):
         if data == b"true":
             return True
-        elif data == b"false":
+        if data == b"false":
```

Comment:
```suggestion
        elif data == b"false":
```

(both should be fine and I'd like to keep the diff minimal)